PhysicsTools/JetMCAlgos/plugins/GenHFHadronMatcher.cc

```diff
@@ -336,7 +340,7 @@ std::vector<int> GenHFHadronMatcher::findHadronJets(const reco::GenParticleColle
                                        topBarDaughterQId,
                                        hadMothersCand,
                                        hadMothersIndices,
-                                       nullptr,
+                                       analyzedParticles,
```

Comment:
This would change the behavior of this call. Previously a new `std::set<const reco::Candidate *>` would be created, Now it gets shared across the loops. To keep the previous behavior would be to create a `std::set<const reco::Candidate *> analyzedParticles` variable in this loop just before the call to `analyzeMothers`.